plugin/view_config/view_config_manager.py

```diff
@@ -119,8 +120,8 @@ def trigger_get_declaration_location(self, view):
         if not config:
             log.debug("Config is not ready yet. No reference is available.")
             return None
-        (row, col) = SublBridge.cursor_pos(view)
-        return config.completer.get_declaration_location(view, row, col)
+        rowcol = ZeroIndexedRowCol.from_1d_location(view, None)
```

Comment:
This should probably be a call to `ZeroIndexedRowCol.from_current_cursor_pos` to make it more readable and to show the intent better.

Reply:
Agreed! Made the change in the latest commit